server/services/discovery/tasks.py

```
@@ -607,7 +678,7 @@ def run_user_discovery(self, user_id):
                 for row in kubectl_rows
             ]
             providers["kubectl"] = {"clusters": clusters}
-            logger.info(f"[Discovery Task] Found {len(clusters)} active kubectl clusters for user {user_id}")
+            logger.info(f"[Discovery Task] Found {len(clusters)} active kubectl clusters for org {org_id}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)